src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 90 Dropdown on Gray 100 background.">
+        <img src="images/Dark_theme_01_new.png" alt="Gray 10 Dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="Gray 80 Dropdown on Gray 90 background.">
+        <img src="images/Dark_theme_02_rev.png" alt="White Dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 70 is sometimes used when compound components are being applied such as an overflow menu.">
+        <img src="images/Dark_theme_03_new.png" alt="White Overflow Menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Do not apply components that are darker than the background unless using high contrast mode.">
+        <img src="images/Dark_theme_01_new.png" alt="Do not apply components that are darker than the background unless using high contrast mode.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Dark theme sample](images/Dark_theme_sample_revised.gif)
+
+_Components with common colors across both dark UI backgrounds_
+
+### High contrast
+
+In some cases, it is helpful to apply light components to dark backgrounds or light components to dark backgrounds. This technique is useful to focus attention or create visual tension when necessary.
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_01.png" alt="High contrast example in a light theme.">    
+    </do-dont-example>
+    <do-dont-example correct=true label=" ">
+        <img src="images/High_contrast_02.png" alt="High contrast example in a dark theme.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+## Interaction states
+
+In addition to the enabled state tokens above, the five other primary interaction states have also been defined and tokenized for each theme. For values between Black and 70, interaction gets lighter and for values between 60 and White, interaction gets darker.
+
+#### Hover
+A unique set of hex values and color tokens exists for the hover states. These colors are outside of the core palette steps and can be considered as half steps in between two adjacent colors. In order to simplify our token structure, the White theme shares the same hover value as the Gray 10 theme; the two dark theme backgrounds share a hover value as well.
+
+#### Active
+The active state is two full steps lighter for values between 100 and 70 and two full steps darker for values between 60 and 10. Exception: White shares the same actived state as Gray 10 and Black shares the same active state as Gray 100. Example: the Blue 60 active state is Blue 80.
```

Comment:
Move the example before the exception. I thought the example was going to be about the exception.